app/models/currency_conversion.rb

```diff
@@ -25,4 +25,15 @@ class CurrencyConversion < ApplicationRecord
   belongs_to :conference
   validates :rate, numericality: { greater_than: 0 }
   validates :from_currency, uniqueness: { scope: :to_currency }, on: :create
+
+  def self.convert_currency(conference, amount, from_currency, to_currency)
+    conversion = conference.currency_conversions.find_by(from_currency: from_currency, to_currency: to_currency)
+    if conversion
+      Money.add_rate(from_currency, to_currency, conversion.rate)
+      amount.exchange_to(to_currency)
+    else
```

Comment:
Maybe consider returning -1 instead instead of returning an amount without conversion. Whenever you use this method convert_currency, you can check for the amount not being -1

Reply:
if you choose this path, remember to update the tests below